andino_gazebo/launch/spawn_robot.launch.py

```diff
@@ -59,13 +59,22 @@ def get_robot_description(use_ros_control: str) -> str:
         os.path.join(
             get_package_share_directory('andino_gazebo'), 'urdf', 'andino.gazebo.xacro'
         ),
-        mappings={'use_gazebo_ros_control': use_ros_control},
+        mappings={
+            'use_gazebo_ros_control': use_ros_control,
+            'use_real_ros_control': 'false',
+            'use_fixed_caster': 'false',
+        },
     )
     robot_desc = doc.toprettyxml(indent='  ')
     folder = get_package_share_directory('andino_description')
     robot_desc = robot_desc.replace(
         'package://andino_description/', f'file://{folder}/'
     )
+    # Solve the problem of the caster
```

Comment:
Can you explain the issue here? This replace is a workaround and as such it would be great if we can explain here why we are doing what we are doing. The alternative would be to create an issue in the repo and add the link here.